shared/profile/index.desktop.js

```diff
@@ -44,20 +44,22 @@ type State = {
   },
 }
 
-const EditControl = ({onClickShowcaseOffer}: {onClickShowcaseOffer: () => void}) => (
+const EditControl = ({isYou, onClickShowcaseOffer}: {isYou: boolean, onClickShowcaseOffer: () => void}) => (
```

Comment:
EditControl is always called with `isYou` set to false, so that param can be removed, right?